src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -14678,29 +14862,82 @@ namespace SurfaceGeometry {
                 ShowContinueError(state, "For explicit details on each use, use Output:Diagnostics,DisplayExtraWarnings;");
             }
         }
-        if (anyGroupedZones) {
-            // All grouped zones have been assigned to an enclosure, now assign remaining zones
-            for (int zoneNum = 1; zoneNum <= state.dataGlobal->NumOfZones; ++zoneNum) {
-                int zoneEnclosureNum = 0;
+        // Check for any spaces defined only by floor surface(s) and group them
```

Comment:
Is a space has only floor surfaces it's lumped into an enclosure for all similar spaces in the same zone plus any "-Remainder" space.